libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/FeatureList/FeatureList.tsx

```diff
@@ -180,7 +180,9 @@ export class FeatureList extends React.Component<
                 placeholder="Search"
                 styles={searchBoxStyles}
                 onSearch={this.onSearch}
-                onClear={this.onSearch}
+                onClear={() => {
+                  this.onSearch("");
```

Comment:
Remove the onClear should fix the problem, btw try not using inline function, as we are going to remove all of them.  Try below, thanks!
```
<SearchBox
                placeholder="Search"
                styles={searchBoxStyles}
                onSearch={this.onSearch}
                onChange={(_, newValue?: string): void => {
                  if (newValue === undefined) {
                    return;
                  }
                  this.onSearch(newValue);
                }}
              />
```

Reply:
thanks, updated.